.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+dist/
```

Comment:
For now you have to commit the dist too. Ref #21

Reply:
why commit dist? I think push dist to npm is enough.

Reply:
Yes, I think the same thing. But there is bower, and to publish the dist 
in a automated we need more workaround. [We are solving this](https://github.com/ApoorvSaxena/lozad.js/pull/21#issuecomment-329359790), but  for now just commit the dist folder.

Reply:
ok，I reverse this config. and this pr can merge?

Reply:
Looks good to me. Let's wait for @ApoorvSaxena to approve.